src/main/resources/templates/gcc/activities-class-hourly-schedule.html

```diff
@@ -1,18 +1,51 @@
+<script th:inline="javascript">
+  function changeVisibiliryActivitiesWeekly() {
+    let locale = $('#activitiesClassHoursSameEveryDay')[0].value
+    let locale = $('#activitiesWeekly')[0].value
+    if
+
+
+    let formAction = [[ ${formAction} ]] + "?lang=" + ("Spanish" === locale ? 'es' : 'en');
+    let form = $('form')[0]
+    form.setAttribute('action', formAction);
+  }
+</script>
 <body>
 <div class="page-wrapper">
   <div th:replace="~{fragments/toolbar :: toolbar}"></div>
   <section class="slab">
     <div class="grid">
       <div th:replace="~{fragments/goBack :: goBackLink}"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
-        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{placeholder}, subtext=#{placeholder})}"/>
+        <th:block th:replace="~{fragments/inputs/headerNameUnderlined :: headerNameUnderlined(text=${fieldData.schoolName})}"></th:block>
+        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{activities-class-hourly-schedule.header}, subtext=#{activities-class-hourly-schedule.description})}"/>
         <th:block th:replace="~{fragments/form :: form(action=${formAction}, content=~{::formContent})}">
           <th:block th:ref="formContent">
             <div class="form-card__content">
-                <!-- Put inputs here -->
+              <div class="question-with-follow-up">
+                <div class="question-with-follow-up__question">
+                  <th:block th:replace="~{fragments/inputs/checkboxFieldset ::
+                                  checkboxFieldset(inputName='activitiesClassHoursSameEveryDay',
+                                  ariaLabel='header',
+                                  content=~{::checkboxOptions})}">
+                    <th:block th:ref="checkboxOptions">
+                      <th:block
+                              th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='activitiesClassHoursSameEveryDay', value='Yes', label=#{activities-class-hourly-schedule.class.hours.same.every.day},toggleVisibility='#activitiesWeekly', followUpId='#same-hours-class-up')}" />
+                    </th:block>
+                  </th:block>
+                </div>
+                <div class="question-with-follow-up__follow-up" id="same-hours-class-up">
+                  <th:block th:replace="~{fragments/inputs/timeRange :: timeRange(startInputName='activitiesClassStartTimeAllDays', endInputName='activitiesClassEndTimeAllDays', label=#{general.week.All})}" />
+                </div>
+              </div>
+              <div id="activitiesWeekly">
+                <th:block th:each="day: ${fieldData['weeklySchedule[]']}">
+                  <th:block th:replace="~{fragments/inputs/timeRange :: timeRange(startInputName='activitiesClassStartTime' + ${day}, endInputName='activitiesClassEndTime' + ${day}, label=#{${'general.week.' + day}})}" />
+                </th:block>
+              </div>
```

Comment:
Somewhere above we can use the checkbox value to set the initial state
```
th:with="isSameEveryDay=${!#lists.isEmpty(fieldData.get('activitiesClassHoursSameEveryDay[]'))}">
```
```suggestion
<div id="grouped-days" th:class="${isSameEveryDay ? '' : 'hidden'}">
                <th:block th:replace="~{fragments/inputs/timeRange ::
                  timeRange(startInputName='activitiesClassStartTimeAllDays',
                    endInputName='activitiesClassEndTimeAllDays',
                    label=#{general.week.All})}"/>
              </div>
```

Reply:
Done.